libcodechecker/analyze/analyzers/result_handler_plist_to_stdout.py

```diff
@@ -91,8 +91,8 @@ def __print_bugs(self, reports, files):
                 LOG.debug(report + ' is skipped (in ' + f_path + ")")
                 continue
 
-            bug = {'hash_value':
-                   report.main['issue_hash_content_of_line_in_context'],
+            bug = {'hash_value': report.main[
```

Comment:
Can we reformat this a bit better? That indent is huge and ugly. Perhaps a local variable and then putting only that into the map?